src/backend/egl/context.rs

```diff
@@ -46,86 +47,67 @@ impl EGLContext {
     ///
     /// - The context must be created from the system default EGL library (`dlopen("libEGL.so")`)
     /// - The `display`, `config`, and `context` must be valid for the lifetime of the returned context.
-    pub unsafe fn from_raw<L>(
+    pub unsafe fn from_raw(
         display: *const c_void,
         config_id: *const c_void,
         context: *const c_void,
-        log: L,
-    ) -> Result<EGLContext, Error>
-    where
-        L: Into<Option<::slog::Logger>>,
-    {
+    ) -> Result<EGLContext, Error> {
         assert!(!display.is_null(), "EGLDisplay pointer is null");
         assert!(!config_id.is_null(), "EGL configuration id pointer is null");
         assert!(!context.is_null(), "EGLContext pointer is null");
-        let log = crate::slog_or_fallback(log.into()).new(o!("smithay_module" => "backend_egl"));
 
-        let display = EGLDisplay::from_raw(display, config_id, log)?;
+        let display = EGLDisplay::from_raw(display, config_id)?;
         let pixel_format = display.get_pixel_format(config_id)?;
 
+        let span = info_span!(parent: &display.span, "egl_context");
```

Comment:
Would it be useful to have a way to identify two different contexts? Probably would be fine to use the pointer for representation? Or some sort of counted I'd?